### ISSUE TITLE
Keep spacing consistent on expanding radio elements

### ISSUE THREAD

**Issue by kauffj** [lbryio/lbry-desktop#679]
On the settings page, spacing changes when selecting radio input rows with additional input (toggle between the price setting fields).

This is not a new issue, but may be more exaggerated after #673

**Comment by alyssaoc** [comment]
Closed, this was fixed in the redesign
